frontend/packages/kubevirt-plugin/integration-tests/tests/vmi.list-filtering.scenario.ts

```diff
@@ -0,0 +1,63 @@
+import { browser } from 'protractor';
+import { appHost, testName } from '@console/internal-integration-tests/protractor.conf';
+import {
+  addLeakableResource,
+  createResource,
+  removeLeakedResources,
+} from '@console/shared/src/test-utils/utils';
+import { isLoaded } from '@console/internal-integration-tests/views/crud.view';
+import { getVMManifest } from './utils/mocks';
+import { VM_STATUS } from './utils/consts';
+import { filterBoxCount } from '../views/vms.list.view';
+import { VirtualMachine } from './models/virtualMachine';
+
+const waitForVM = async (
+  manifest: any,
+  status: VM_STATUS,
+  resourcesSet: Set<string>,
+  kind?: 'virtualmachines' | 'virtualmachineinstances',
+) => {
+  const vm = new VirtualMachine(manifest.metadata, kind || 'virtualmachines');
+
+  createResource(manifest);
+  addLeakableResource(resourcesSet, manifest);
+  await vm.waitForStatus(status);
+};
+
+const waitForVMList = async () => {
+  await browser.get(`${appHost}/k8s/ns/${testName}/virtualmachines`);
+  await isLoaded();
+};
+
+describe('Test List View Filtering (VMI)', () => {
```

Comment:
not sure if the naming applies ^^, we are not filtering anything - we show all of the VMs and VMIs at once

Reply:
> we are not filtering anything

We are filtering by status, we have 1 `Off` vm and 1 `Running` vmi,
A side affect of this filtering procedure is the we know for sure the we list both vm and vmi in one list.

Reply:
I mean the name implies that we are testing the filtering feature of the UI. We are just reading the values and not testing it

Reply:
Correct, we are actually testing [vmStatusFilter](https://github.com/openshift/console/blob/master/frontend/packages/kubevirt-plugin/src/components/vms/table-filters.ts) that filter the vm/vmi list, and not the actual filtering feature of the UI , implemented in `MultiListPage`.

This actually makes sense here because the UI filtering is a core feature that we should not test in our module, maybe we should rename the test to make it clear we test `vmStatusFilter` and not `MultiListPage` that does the UI filtering ?

@rhrazdil @suomiy WDYT ?

Reply:
It's true that name like: `Displays correct numbers of Running VMIs and Off VMs` or similar would probably better describe what is done here.
I would keep the test names e2e oriented from user's perspective. IMO there is no need to make distinction betweem our plugin vs core in the test cases names.